dspace/config/crosswalks/DIM2DataCite.xsl

```diff
@@ -50,12 +53,20 @@
             <!-- 
                 DataCite (1)
                 Template Call for DOI identifier.
-            --> 
-            <xsl:apply-templates select="//dspace:field[@mdschema='dc' and @element='identifier' and starts-with(., 'http://dx.doi.org/')]" />
+                Occ: 1
+            -->
+            <!--
+                dc.identifier.uri may contain more than one DOI, e.g. if the
+                repository contains an item that is published by a publishing 
```

Comment:
My understanding is that dc.identifier.uri is for identifiers minted by DSpace itself, and that identifiers supplied manually should be in dc.identifier.  So there may be any number of values for dc.identifier.uri but only one DOI among them.